### PR TITLE
[OS-4391] Update repo to deploy new PyPi package

### PR DESCRIPTION
**Pull request by ojc-orchestra**
# Changes

- update `.gitignore`
- remove old `.idea` folder
- update LICENSE
- add a Makefile with common development commands
- update README
- change setup.py files to more modern `pyproject.toml` structure

No application code updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,56 @@
...
-The basic premise is for your self-hosted application to be triggerable via Orchestra, so accepts a
-webhook or HTTP request. Within the application, the orchestra-logger is added to send checkpoint information
-to orchestra, such as whether the request was successful, whether it failed, why etc. That's it!
+This is a lightweight SDK that allows Orchestra ([getorchestra.io](https://www.getorchestra.io/)) to interact with self-hosted applications (Tasks).
 
-## Quickstart - local testing
+The basic premise is for your self-hosted Task to send back status updates and logs to Orchestra. This is done via a HTTP request. The Task is started by Orchestra.
 
-Firstly, run 
+## Local Development
```
Comment (amyc92) [inline README.md:9]: assume this stuff will change and i can ignore?

File: src/orchestra_sdk/orcProcess/orcProcess.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import inspect
+from datetime import datetime
+
+from orchestra_sdk.orchttp.orchttp import orcHTTP
+
+
+class orcProcess:
```
Comment (amyc92) [inline src/orchestra_sdk/orcProcess/orcProcess.py:7]: assume i can ignore code stuff?

**Review by amyc92 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ojc-orchestra) [reply README.md:9]: Everything under `Quickstart guide` can be ignored - the local development bit I think will probably stay as it is now

File: src/orchestra_sdk/orcProcess/orcProcess.py (hunk 2 above)
Comment (ojc-orchestra) [reply src/orchestra_sdk/orcProcess/orcProcess.py:7]: indeed

File: README.md (hunk 1 above)
Comment (amyc92) [reply README.md:9]: not sure we should publish local development details on the package readme unless we are going to open source this?

Comment (ojc-orchestra) [reply README.md]: fair - I have moved those instructions to a different README inside of `src/`

**Review by amyc92 — APPROVED** [review]

**Pull request merged**